pkg/jmxfetch/jmxfetch.go

```diff
@@ -267,3 +285,71 @@ func (j *JMXFetch) Up() (bool, error) {
 	err = process.Signal(syscall.Signal(0))
 	return err == nil, err
 }
+
+func (j *JMXFetch) ConfigureCheck(initConfig integration.Data) error {
```

Comment:
this function is exported, so the go linter should require a small docstring on top of it, not sure why it's not catching it. Can you add one?

Also, I'd prefer a name like `ConfigureFromInitConfig` to reflect that we're configuring the `JMXFetch` struct from what's in `initConfig`, which sounds more explicit to me than `ConfigureCheck`. WDYT?

Reply:
Good idea, changed.